onnxruntime/test/onnx/microbenchmark/quantize_linear.cc

```diff
@@ -0,0 +1,80 @@
+#include "common.h"
+
+#include <benchmark/benchmark.h>
+#include <core/util/math_cpuonly.h>
+#include <core/mlas/lib/mlasi.h>
```

Comment:
I don't like that we are including an internal implementation header from this test directory. Is this benchmark really necessary to include? If you want a benchmark that just calls the public MlasQuantizeLinear, then I guess that's okay.

Reply:
i meant to compare the performance between avx512 and sse. Let me change it and benchmark MlasQuantizeLinear only.